build.gradle

```diff
@@ -34,4 +34,6 @@ ext {
     junitVersion = '4.12'
     mockitoVersion = '1.10.19'
     assertjVersion = '2.5.0'
+
+    lintVersion = '25.4.0-alpha7'
```

Comment:
The latest stable release of the Lint libraries (25.3.0) doesn't include UAST yet, no. I suppose it'll be out once AS 3.0 is stable, though.